components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -223,11 +240,11 @@ export default class TeamSelectorModal extends React.PureComponent {
 
         const buttonSubmitText = localizeMessage('multiselect.add', 'Add');
 
-        let teams = [];
+        let teams = [] as Array<Team>;
         if (this.props.teams) {
             teams = this.props.teams.filter((team) => team.delete_at === 0);
             teams = teams.filter((team) => team.scheme_id !== this.currentSchemeId);
-            teams = teams.filter((team) => this.props.alreadySelected.indexOf(team.id) === -1);
+            teams = teams.filter((team) => this.props?.alreadySelected?.indexOf(team.id) === -1);
```

Comment:
I dont think we need optional chaining for props. We can used `this.props` directly.

Reply:
done in latest commit